src/main/java/com/battleship/backend/models/Board.java

```diff
@@ -66,6 +66,18 @@ public boolean hasShip(Ship maybeShip){
         return false;
     }
 
+    public boolean shipListEquals(ArrayList<Ship> otherShipList){
```

Comment:
Nit: could you please make this adhere to Java's naming convention for booleans?

Reply:
Yes, changed to isShipListEquals.

Reply:
Actually, changed to isPermutationOfShipList